packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3017,6 +3017,33 @@ Specifies the profile for a `question` factor
 }
 ```
 
+#### Question Profile - Unique Keys for question
+
+Referenced from [okta-auth-js questions](https://github.com/okta/okta-auth-js/blob/master/test/xhr/questions.js)
```

Comment:
Let's drop this in favor of something like:
> The keys for the built-in security questions are listed below.

Reply:
Made the edit.